src/main.js

```diff
@@ -42,7 +43,10 @@ Vue.component('VSelect', VueSelect);
 Vue.component('VueGriddle', VueGriddle);
 Vue.component('ValidationProvider', ValidationProvider);
 Vue.component('ValidationObserver', ValidationObserver);
+Vue.component('FormSection', FormSection);
 Vue.component('FormComponent', FormCompoent);
+Vue.component('BtnSave', BtnSave);
+Vue.component('BtnDelete', BtnDelete);
```

Comment:
Hvor legger vi lista for hva som skal være globale komponenter? Betyr det noe mer enn at de kan brukes overalt uten at man trenger å importere dem?

Reply:
Usikker. Så at `ValidationObserver`, `-Provider` og `FormComponent` allerede er globale komponenter, og i og med at disse generelle skjema-komponentene også blir hyppig brukt, er det jo kanskje ikke så rart, om enn først og fremst "convenience". Tenker for øvrig at `Validation*`-komponentene kan fjernes som globale i den grad vi primært bruker `FormSection` og `FormComponent`.

[Globale komponenter](https://v2.vuejs.org/v2/guide/components-registration.html#Global-Registration) påvirker vel potensielt "bundle size" og ytelse:
> Global registration often isn’t ideal. For example, if you’re using a build system like Webpack, globally registering all components means that even if you stop using a component, it could still be included in your final build. This unnecessarily increases the amount of JavaScript your users have to download.

Imidlertid usikker på hvorvidt dette forholder seg anerledes med Vite/Rollup vs. Webpack osv., men i alle tilfeller en avveining. Åpen for andre meninger om det.

Reply:
Ja, det er jo kanskje vanskeligere å huske å slette globale komponenter som ikke lenger er i bruk enn om de var lokalt importert. Og så synes jeg det er lettere å se hvilke komponenter som er i bruk og hvor de kommer fra når de importeres lokalt. Får litt sånn `from x import *` følelse av de globale. :sweat_smile:

Reply:
Ja, kan godt jobbe for å fjerne globale komponenter etter hvert som vi kommer over dem - og i dette tilfellet ikke innføre nye.